src/bld/winCfg.js

```diff
@@ -34,13 +37,24 @@ const setWinConfig = async (app, outDir) => {
     }
   });
 
-  await rename(`${outDir}/nw.exe`, `${outDir}/${app.name}.exe`);
-  await rcedit(`${outDir}/${app.name}.exe`, {
-    "file-version": app.version,
-    "icon": app.icon,
-    "product-version": app.version,
-    "version-string": versionString,
-  });
+  try {
+    await rename(`${outDir}/nw.exe`, `${outDir}/${app.name}.exe`);
+
+    if (platform === "win32") {
```

Comment:
Isn't `process.platform` going to not equal "win32" when running in macOS or Linux with WINE? I don't think this check will work in those operating systems.

I think you can probably get away with something like:

```js
try {
    await rcedit(...);
} catch (error) {
    throw new Error(`Unable to modify EXE. Ensure WINE is installed or build in Windows.\nError: ${error}`);
}
```

Reply:
I didn't `throw` the error since the error is not caught in the parent function.